doc/api/tls.markdown

```diff
@@ -162,7 +162,10 @@ automatically set as a listener for the [secureConnection][] event.  The
     the `key`, `cert` and `ca` options.)
 
   - `key`: A string or `Buffer` containing the private key of the server in
-    PEM format. (Could be an array of keys). (Required)
+    PEM format. It can also be an array of keys. The array can either be of
+    just keys or if you have different passphrases for the keys, then the
+    array elements can be of the form `{pem: key, passphrase: passphrase}` and
+    the keys should use different algorithms. (Required)
```

Comment:
@silverwind Hmmm, it was @indutny's suggestion. Let's see what he feels about this.